sdk/loadtestservice/Azure.Developer.LoadTesting/src/autorest.md

```diff
@@ -6,22 +6,62 @@ Run `dotnet build /t:GenerateCode` to generate code.
 namespace: Azure.Developer.LoadTesting
 security: AADToken
 security-scopes: https://cnt-prod.loadtesting.azure.com/.default
 skip-csproj-packagereference: true
 directive:
-- from: swagger-document
-  where: '$.paths.*[?(@.tags=="AppComponent")]'
-  transform: >
-    $["operationId"] = $["operationId"].replace("AppComponent_", "LoadTestAdministration_");
-- from: swagger-document
-  where: '$.paths.*[?(@.tags=="ServerMetrics")]'
-  transform: >
-    $["operationId"] = $["operationId"].replace("ServerMetrics_", "LoadTestAdministration_");
-- from: swagger-document
-  where: '$.paths.*[?(@.tags=="Test")]'
-  transform: >
-    $["operationId"] = $["operationId"].replace("Test_", "LoadTestAdministration_");
+    - from: swagger-document 
+      where: $["paths"]["/tests/{testId}"].patch
+      transform: $["operationId"] = "Test_CreateOrUpdateTest";
+    - from: swagger-document 
+      where: $["paths"]["/tests/{testId}"].delete
+      transform: $["operationId"] = "Test_DeleteTest";
+    - from: swagger-document 
+      where: $["paths"]["/tests/{testId}"].get
+      transform: $["operationId"] = "Test_GetTest";
+    - from: swagger-document 
+      where: $["paths"]["/tests"].get
+      transform: $["operationId"] = "Test_ListTests";
+    - from: swagger-document 
+      where: $["paths"]["/test-runs/{testRunId}/metric-dimension/{name}/values"].get
+      transform: $["operationId"] = "TestRun_ListMetricDimensionValues";
+    - from: swagger-document 
+      where: $["paths"]["/tests/{testId}/files/{fileName}"].put
+      transform: $["operationId"] = "Test_UploadTestFile";
+    - from: swagger-document 
+      where: $["paths"]["/tests/{testId}/files/{fileName}"].get
+      transform: $["operationId"] = "Test_GetTestFile";
+    - from: swagger-document 
+      where: $["paths"]["/tests/{testId}/files/{fileName}"].delete
+      transform: $["operationId"] = "Test_DeleteTestFile";
+    - from: swagger-document 
+      where: $["paths"]["/tests/{testId}/files"].get
+      transform: $["operationId"] = "Test_ListTestFiles";
+    - from: swagger-document 
+      where: $["paths"]["/test-runs/{testRunId}"].delete
+      transform: $["operationId"] = "TestRun_DeleteTestRun";
+    - from: swagger-document 
+      where: $["paths"]["/test-runs/{testRunId}"].patch
+      transform: $["operationId"] = "TestRun_CreateOrUpdateTestRun";
+    - from: swagger-document 
+      where: $["paths"]["/test-runs/{testRunId}"].get
+      transform: $["operationId"] = "TestRun_GetTestRun";
+    - from: swagger-document 
+      where: $["paths"]["/test-runs/{testRunId}/files/{fileName}"].get
+      transform: $["operationId"] = "TestRun_GetTestRunFile";
+    - from: swagger-document 
+      where: $["paths"]["/test-runs"].get
+      transform: $["operationId"] = "TestRun_ListTestRuns";
+    - from: swagger-document 
+      where: $["paths"]["/test-runs/{testRunId}:stop"].post
+      transform: $["operationId"] = "TestRun_StopTestRun";
```

Comment:
Why are we doing all these renames? Why can't we use the operationIds directly from the REST API?